### PR TITLE
#155922722 import asset categories and subcategories into the system

### PR DESCRIPTION
**Pull request by QUDUSKUNLE**
### What does this PR do?
- It allows users to import asset categories and subcategories into the system

### Description of Task to be completed?
- Write an import script to import categories and subcategories into the system
- Display summary of the operation

### How should this be manually tested?

- `run python category_import.py`

### Screenshot
- 
screen shot 2018-06-09 at 10 25 25 am
screen shot 2018-06-09 at 10 27 05 am
screen shot 2018-06-09 at 10 27 25 am

### CODE REVIEW

File: category_import.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import sys
+import os
+import django
+import pandas as pd
```
[the remark this answers is not in the document]
Comment (QUDUSKUNLE) [reply category_import.py]: Alright, I would look into that.

File: category_import.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+project_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(project_dir)
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings")
+django.setup()
+
+from core.models.asset import AssetCategory, AssetSubCategory  # noqa
+
+
+def create_category(x):
```
[the remark this answers is not in the document]
Comment (QUDUSKUNLE) [reply category_import.py]: Noted

File: category_import.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        )
+        add_sub_category.save()
+        return 'Sub category created'
+
+
+def csv_file(file):
+    if os.path.isfile(file):
+        if not file.endswith('.csv'):
+            return 'File not supported'
+        else:
```
[the remark this answers is not in the document]
Comment (QUDUSKUNLE) [reply category_import.py]: Noted

File: category_import.py [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+                        counter += 1
+                        pbar.update(1)
+            print("\n")
+            display_inserted(inserted_records)
+            display_skipped(skipped)
+
+    return 'File not found'
+
+
+def display_inserted(result):
```
Comment (andela-mabdussalam) [inline category_import.py]: @QUDUSKUNLE We now have this in a separate file. Pls import from https://github.com/AndelaOSP/art-backend/blob/develop/import_util.py

File: category_import.py [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+    if len(result) <= 0:
+        print("No record was inserted\n")
+    else:
+        print('{0} \t{1} \t '.format('Line No.', 'Description'))
+        print('------------------------------------------------------------\n')
+        for item in result:
+            print('{0}\t{1}'.format(item[1], item[0]))
+
+
+def display_skipped(result):
```
Comment (andela-mabdussalam) [inline category_import.py]: Same as above.

File: category_import.py (hunk 4 above)
Comment (QUDUSKUNLE) [reply category_import.py]: Fixed.

File: category_import.py (hunk 5 above)
Comment (QUDUSKUNLE) [reply category_import.py]: Changes requested had been fixed.

Comment (QUDUSKUNLE) [reply category_import.py]: changes effected

**Comment by egichuri** [comment]
@QUDUSKUNLE please merge this

**Pull request merged**